docs/basics/getting-started.md

```diff
@@ -1,210 +1,159 @@
-in this documentation that are not present in the migration guide are still valid.
+This software is designed primarily for **private trackers**. If you plan on
+cross-seeding on public trackers,
+[please see our FAQ](./faq-troubleshooting.md#does-cross-seed-work-on-public-trackers)
+for more details.
 
-If you have any questions please feel free to contact us on Discord or the Discussions section of GitHub. There are links located as
-icons in the top right of every page.
 :::
 
-:::danger
-Certain `cross-seed` configurations can result in downloading an excessive number of torrent files in a short amount of time.
-Trackers may, as a result, forbid this or require specific settings to minimize this behavior as much as possible.
+---
 
-[See this FAQ post for details](./faq-troubleshooting.md#my-tracker-is-mad-at-me-for-snatching-too-many-torrent-files)
-:::
+## Setting Up the Daemon
+
+The most efficient way to use `cross-seed` is to run it in **daemon mode**,
+which continuously monitors for new downloads and cross-seeds them
+automatically. Here’s how to get everything up and running.
 
-<hr/>
-Now we will install `cross-seed` and run our first search.
+### 1. Install `cross-seed`
 
-## Installation
+You can install `cross-seed` using one of the following methods:
 
-### with `npm` (recommended if you're not already using Docker)
+#### With `npm` (recommended if you're not using Docker)
 
-Requires [node](https://nodejs.org/en/download/) 16 or greater.
+You’ll need **Node.js 20** or greater.
 
 ```shell
 npm install -g cross-seed
 cross-seed --version
 ```
 
-### with Docker
+#### With Docker
 
 ```shell
 docker run ghcr.io/cross-seed/cross-seed --version
 ```
 
-### with Unraid
-
-In Unraid, you can install `cross-seed` from Docker Hub with the Community
-Applications app. Check out the [Unraid guide](../tutorials/unraid) for details.
-
-## Running your first search
+#### With Unraid
+For Unraid installation, see the [Unraid guide](../tutorials/unraid.md).
 
-:::tip
+---
 
-Docker users can skip ahead to [Scaling Up](#with-docker-1).
+### 2. Create a Config File
 
-:::
+You’ll need to generate a configuration file before running the daemon. Both
+Docker and non-Docker users will use the same method for this.
 
-To get started, you can use CLI.
+#### For non-Docker users:
 
-If you already have a `config.js` file set up, then you can simply run the following to use those settings:
+Run the following command to generate a config file:
 
 ```shell
-cross-seed search
+cross-seed gen-config
 ```
 
-:::tip
-Any CLI argument provided will override the settings used in your `config.js`
-:::
+This will create a basic `config.js` file at `~/.cross-seed/config.js` or
+`AppData\Local\cross-seed\config.js`. You can then edit this file to add the
+necessary configurations.
+
+#### For Docker users:
 
-For CLI commands, a basic example is below:
+Docker users can generate the config file by running the following command:
```

Comment:
IIRC - Upon running it without a config, one will be generated. So you don't really need to specify that running the gen-config command is needed explicitly.

Perhaps mentioning "When you first run cross-seed, if no configuration options are set and a configuration file does not exist, one will be generated for you. Alternatively, you can run the gen-config command to generate one." If this is something you feel needs to be specifically mentioned.